### DISCUSSION TITLE
Hot reload broken

### DISCUSSION THREAD

**Discussion by onurusluca** [nuxt/nuxt#27646] [Questions]
2 days ago suddenly my Nuxt app's hot reload broke.  I don't remember doing anything out of the ordinary. I only update the packages normally but I don't remember if I updated before this problem. I don't know what version I need to go back. 
This doesn't happen with a new project. Can't figure out what the problem is.

Getting this error:
`GET http://localhost:24678/_nuxt/ net::ERR_CONNECTION_REFUSED`

Tried these:
- Restarted the server, browser, pc, router
- Deleted, .nuxt, .outpu, node_modules, lock file and reinstalled again
- Tried 3 different browsers, all same
- Tried with a fresh installed browser, no extensions etc, Same
- Turned VPN on/off, same. Uninstalled it, restarted the pc. Same

Here is some info:
------------------------------
```
- Operating System: Windows_NT
- Node Version:     v20.11.1
- Nuxt Version:     3.9.3
- CLI Version:      3.12.0
- Nitro Version:    2.9.6
- Package Manager:  pnpm@9.1.1
- Builder:          -
- User Config:      devtools, app, routeRules, $production, $development, runtimeConfig, vite, modules, supabase, tailwindcss, pinia, i18n, shadcn, colorMode, googleFonts
- Runtime Modules:  @nuxtjs/supabase@1.2.2, @pinia/nuxt@0.5.1, @nuxtjs/i18n@8.3.1, @nuxtjs/tailwindcss@6.12.0, shadcn-nuxt@0.10.4, nuxt-icon@0.6.10, @nuxtjs/color-mode@3.4.1, @nuxt/image@1.7.0, @nuxtjs/google-fonts@3.2.0, @nuxt/eslint@0.3.13
- Build Modules:    -
```

**Comment by onurusluca** [comment]
Just as I was writing this, I manually updated to `nuxt 3.12.1` and the problem is gone.

**Reply by AloisSeckar** [reply]
I had this issue few days ago and figured out it is comming from latest `vite`. I switched back to older `5.2.10` and it also helped. But it was  before Nuxt 3.12, so now I will try to upgrade Nuxt as well and see.
